### DISCUSSION TITLE
Class 'chillerlan\QRCode\QROptions' not found Codeigniter 3

### DISCUSSION THREAD

**Discussion by Mobin-khorushi** [chillerlan/php-qrcode#175] [Q&A]
Hi im using Codeigniter3 and i wanted to use this library which i installed via composer in my local and pushed all to my server 
(Same way im doing with all of my vendor )
Everything working well expect when i include QROptions i get this errors :

`
Class 'chillerlan\QRCode\QROptions' not found
`
PHP Version is 7.2
im running cPanel on my server
"chillerlan/php-qrcode": "^4.3"

QROptions.php is there also i enabled my autoload & loaded manually both ways i get this error

**Comment by codemasher** [comment]
Hi, first off version 4.x won't work with PHP 7.2, you will need to install `^3.4` instead. Second, this still sounds like there is an issue with the autoloader?
